### PR TITLE
feat(ai task): adds info about knowledge cutoff

### PR DESCRIPTION
**Pull request by devguilhermy**

### CODE REVIEW

File: pages/integration/howTo/_meta.json
Comment (ptrckbp) [inline pages/integration/howTo/_meta.json:1]: This might be the issue causing the preview to fail

Comment (devguilhermy) [reply pages/integration/howTo/_meta.json:1]: It was the new Date() code, it had to `` around it

File: pages/cloud/toolbox/ai-task.mdx [hunk 1]
```diff
@@ -75,9 +81,8 @@ to solve real-world problems and generate useful content. This can ultimately le
 for users who are targeting the power of artificial intelligence in their workflows.
 
 <br />
-The Task Example allows you to provide example text that the AI Task can expect to
-receive from the user, as well as examples of the outputs the AI Task should generate
-as a response. These examples help the AI Task to better understand the instruction
+The Task Example allows you to provide example text that the AI Task can expect to receive from the user, as well as examples
```
Comment (ptrckbp) [inline pages/cloud/toolbox/ai-task.mdx:84]: next time you can push this to another PR, as this is different content. :)

File: pages/integration/howTo/build-integration.mdx [hunk 2]
```diff
@@ -24,7 +23,7 @@ This guide will primarily focus on integrating with ClickUp, a highly-respected
 
 Kickstart your journey by watching our comprehensive video series on building a ClickUp integration with Botpress. Our in-house experts walk you through the entire process, from the initial setup to handling outgoing messages.
 
---- 
+---
```
Comment (ptrckbp) [inline pages/integration/howTo/build-integration.mdx:26]: Not super important, but we can simplify our PRs without these

**Review by ptrckbp — APPROVED** [review]

**Pull request merged**